### PR TITLE
Supplemental docs PR to GH-35 of LocalDeployer

### PR DESCRIPTION
**Pull request by olegz**
This PR simply documents remote debugging with LocalDeployer

### CODE REVIEW

**Comment by olegz** [comment]
NOTE: This PR must be merged *ONLY* after https://github.com/spring-cloud/spring-cloud-deployer-local/pull/85

File: spring-cloud-dataflow-docs/src/main/asciidoc/appendix-howto.adoc [hunk 1]
```diff
@@ -168,6 +168,24 @@ These properties can also be specified via `deployment` properties when deployin
 dataflow:>stream deploy foo --properties "app.*.logging.level.org.springframework.integration=DEBUG"
 ----
 
+==== Remote Debugging with Local Deployer
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/appendix-howto.adoc]: Can we rephrase this to: Remote Debugging with Local Server"?

File: spring-cloud-dataflow-docs/src/main/asciidoc/appendix-howto.adoc [hunk 2]
```diff
@@ -168,6 +168,24 @@ These properties can also be specified via `deployment` properties when deployin
...
+
+When using Local Deployer you may want to benefit from the ability to debug your individual applications. This could be accomplished using remote debugging feature of the JVM.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/appendix-howto.adoc]: Same here, too. Language-wise, the SCDF users don't have to know the support in the deployer; the user-facing support is in SCDF (via deployer) anyway.

**Comment by markpollack** [comment]
minor edits merged as f4ef6abd3112ef71dcd9be8371c70de8b309c7d5
